### PR TITLE
Feat/update bp info with standard 863

### LINKED ISSUE #836: Fix broken images of block producers
There are some block producers whose thumbnails are broken, and this makes the page look noisy. Is it possible to define an image that replaces any broken thumbnail?

### LINKED ISSUE #863: Update bpJson fiels and nodes acording with bpjson standers information
There are new fields on bpJson
lest check that all the new information is in node fields too.

- [x] Check missing fields according to standard
- [x] Add the missing fields
- [x] Refactor
- [x] #836
- [x] #874

### LINKED ISSUE #874: Find the logo for the diferent nodos
lest find the logo for the diferents nodos producer and update the table 

image

### PR DESCRIPTION
**Pull request by Torresmorah**
### What does this PR do?

- Resolves #863
- Resolves #836 
- Resolves #874
- Refactor related components
- Hide "More Info" button on desktop

### Steps to test

1. Run the project locally
2. Check the paths of block producers and nodes

#### CheckList

- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]
LGTM

File: webapp/src/components/NodeCard/index.js [hunk 1]
```diff
@@ -24,31 +23,44 @@ const useStyles = makeStyles(styles)
 const NodeCard = ({ producer, node }) => {
   const classes = useStyles()
   const { data: { cpu } = {} } = useQuery(NODE_CPU_BENCHMARK, {
-    variables: { account: node?.name || producer.owner }
+    variables: { account: node?.name || producer.owner },
   })
   const { t } = useTranslation('nodeCardComponent')
   const [producerOrg, setProducerOrg] = useState({})
 
+  const ShowInfo = ({ cond, title, value }) => {
```
Comment (tetogomez) [inline webapp/src/components/NodeCard/index.js:31]: single return and space after if declaration

```
const ShowInfo = ({ cond, title, value }) => {
    if (!cond && !value)  return <></>
    
    return (
      <>
        <dt className={classes.bold}>{title}</dt>
        <dd>{value}</dd>
      </>
    )
  }

File: webapp/src/components/ProducerCard/index.js [hunk 2]
```diff
@@ -35,11 +35,13 @@ const ProducerCard = ({ producer, onNodeClick, rank }) => {
   }, [producer])
 
   const Avatar = () => {
+    const logo = producerOrg.branding?.logo_256
```
Comment (tetogomez) [inline webapp/src/components/ProducerCard/index.js:38]: nice!

File: webapp/src/utils/validate-image.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const isLogoValid = (imgUrl) => {
```
Comment (tetogomez) [inline webapp/src/utils/validate-image.js:1]: space after `const or let` declaration and space bofer return

```
const isLogoValid = (imgUrl) => {
  const img = new Image()
  
  img.src = imgUrl
  
  return img.width >= 100 && img.height >= 100
}

**Review by tetogomez — CHANGES_REQUESTED** [review]
just a couple of comments

BTW: good job removing Box components

**Pull request merged**